README.md

```diff
@@ -1,58 +1,83 @@
 # Satellite Link Example
 
-### 1. Deploy the application on Red Hat OpenShift cluster
-
-#### 1.1. Connect to the OpenShift cluster in your CLI
-
-- To log in, you need a token which can be generated after you log in to OpenShift Cluster web console. Click the drop-down under your name in the right corner of your screen and select Copy Login Command.
-![](images/redhat1.png)
-
-- A new window will open requesting the login token details. Copy the login token. Copy the login token
-![](images/redhat2.png)
-
-- In your terminal, paste the login token. Run the command.
-
-- You are now connected to your OpenShift cluster.
-
-#### 1.2. Create a new project as follows,
-
-    oc new-project <project_name>
-
-#### 1.3. Download the `ExampleApp/deployment.yaml` in this repo, and run the following command
-
-    oc  apply -f  deployment.yaml
-
-#### 1.4. Run the following command and copy the route url 
-
-    oc get routes -n <project_name>
-    
-![](images/oc.png)
-
-### 2. Run the application
-
-#### 2.1 Fill in the details as shown below.
-
-In place of host url and port, give your satellite endpoint url and its port. All other details will be found in the credential section of the service.
+This simple application allows to query a PostgreSQL database. The form prompts you for the database credentials. These credentials will be sent to the application running in the cluster and the connection will be made to the database over Satellite link.
+
+This sample requires:
+- a running Satellite location,
+- an OpenShift cluster running in the location with its image registry correctly configured,
+- a _Databases for PostgreSQL_ instance running in IBM Cloud.
+
+## Deploy the application on Red Hat OpenShift cluster
+
+### Connect to the OpenShift cluster in your CLI
+
+1. To log in, you need a token which can be generated after you log in to OpenShift Cluster web console. Click the drop-down under your name in the right corner of your screen and select Copy Login Command.
+   ![](images/redhat1.png)
+1. A new window will open requesting the login token details. Copy the login token. Copy the login token
+   ![](images/redhat2.png)
+1. In your terminal, paste the login token. Run the command.
+1. You are now connected to your OpenShift cluster.
+
+### Create the application
+
+1. Create a new project:
+   ```sh
+   oc new-project <project_name>
+   ```
+1. Create a new application in the OpenShift project:
+   ```sh
+   oc new-app python~https://github.com/l2fprod/satellite-link-example.git --name link-example
```

Comment:
change https://github.com/l2fprod/satellite-link-example.git to https://github.com/IBM/satellite-link-example.git